Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shuttle"
-version = "0.4.1"
+version = "0.5.0"
```

Comment:
Could you undo this? I'll prep a release in a separate commit since I need to do the changelog too.

Reply:
Never mind, forgot that GitHub lets me do this myself.

Reply:
ah sorry, next time I'll be more careful and will not touch this